src/components/RowMarker.tsx

```diff
@@ -19,6 +21,9 @@ const RowMarker = (props: Types.RowMarkerProps): JSX.Element => {
   const markerHighlight = highlightValidRow(true);
   const getVisibleClass = props.isVisible ? "" : "invisible";
   const numberOfRows = givenRow.rows.length;
+  
+  
+
```

Comment:
Let's remove unnecessary new lines: you can see lots of unnecessary new lines like this